SMPyBandits/Environment/MAB_rotting.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
It has a completely different API than the other `MAB.py` file, but if it works…

Reply:
I know ! I don't think it is better but I think I will not have time to go through your API, understand it, and refactor my existing code.